### DISCUSSION TITLE
Add-in Button in web version

### DISCUSSION THREAD

**Discussion by antonycabero** [microsoft/MHA#309] [Q&A] [answered]
Hello @stephenegriffin, 

I'm developing an Outlook add-in and I would like to know what file and what code lines are related to your button position in web version ? My button appears only in the "Other actions" button menu but doesn't show like your.

I guess it's in the manifest but even if I'm trying to have a similar file, my button doesn't show in this menu. How did you do ? 

![MHA button outlook](https://user-images.githubusercontent.com/28923514/131506512-1a2841fb-01a5-442f-9b20-6c3fe2f89dd8.PNG)

Thanks in advance,
Antony C.

**Comment by stephenegriffin** [comment, marked as answer]
Pretty sure it's just an [add-in command](https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/add-in-commands-for-outlook). See this [demo](https://github.com/officedev/outlook-add-in-command-demo).
